crpc/rpc.proto

```diff
@@ -80,6 +80,18 @@ service Connector {
     //
     // Estimate estimates the dollar price of the choosen asset.
     rpc Estimate (EstimateRequest) returns (EstimationResponse);
+
+    //
+    // PaymentReceived is used to determine if payment with given ID is received
+    rpc PaymentReceived (PaymentReceivedRequest) returns (PaymentReceivedResponse);
+}
+
+message PaymentReceivedRequest {
+    string id = 1;
+}
+
+message PaymentReceivedResponse {
```

Comment:
Shouldn't we give more details about payment there?

Reply:
What kind of details? I think this is enough.